docs/articles/new-expensify/get-paid-back/Distance-Requests.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Distance Requests
+description: How to create a distance request and request reimbursement for mileage
+---
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# Overview
+
+Expensify allows you to request reimbursement for mileage by creating a distance request from a map. You can send a distance request in Expensify's mobile app, desktop app, or web app.
+
+
+# How to create and send a distance request
+
+1. Click the green '+' button and select Request Money.
+2. Select Distance along the top row of the Request Money window.
+3. Enter the Start and Finish addresses then click Next. If there are multiple stops, you can add them before clicking Next.
+4. Select your organization's workspace from the list of recent workspaces.
+5. On the confirmation page, confirm the amount, Date, and Distance, and optionally enter a Description and Category. Click the Request button.
+6. A workspace admin will receive your request and can reimburse you through Expensify or elsewhere!
+
+
+
+# FAQs
+
+
+## How can I change the mileage rate used on my distance requests or add an additional rate?
+
+If you use a group workspace, then you can change the rate in your Workspace settings. In Expensify Classic, go to Settings > Workspaces > [Your Workspace] > Expenses > Distance > Add a Mileage Rate. If you submit mileage expenses on a group workspace, you must be a workspace admin to make changes.
```

Comment:
Free workspaces use the default mileage rate which can be modified in NewDot > Settings > Reimbursements. Paid group workspaces can edit the default distance rate in NewDot or OldDot. They can also add rates in OldDot, but we don't have a way to select other rates yet. We'll add it in the P2P project. I think we should leave out the individual part until we finish building P2P distance requests.

Reply:
Good call.

Reply:
To be clear, and for simplicity, I think we should only answer a question about how to change the default mileage rate on NewDot, via NewDot > Settings > Reimbursements. You can't select an additional rate in NewDot right now so we shouldn't discuss that yet.

Reply:
Done 👍

Reply:
I meant the question should be "How can I change the rate use for distance requests" and the answer "Go to Settings > Workspaces > Select your workspace > Reimbursements. Click on the rate to adjust it and the unit, then hit Save. Future requests will use the updated rate.